pkgs/games/uqm/default.nix

```
@@ -65,6 +65,12 @@ in stdenv.mkDerivation rec {
     ln -s "${videos}" "uqm-${version}/content/addons/3dovideo"
   '';
 
+  patchFlags = [ "-p0" ];
+  patches = [ ./tempfile.patch ];
```

```suggestion
  postPatch = "substituteInPlace build/unix/config_functions --replace "/tmp/build.$$.tmp" "$TMP/build.$$.tmp"
```
An inline one-line patch is preferable IMHO vs a separate patch file which might need removed in future.